lib/hydra/works/services/determine_original_name.rb

```diff
@@ -0,0 +1,15 @@
+module Hydra::Works
+  class DetermineOriginalName
+    # Determines the original name for a given file
+    # @param [IO, File, Rack::Multipart::UploadedFile, #read] file
+    # @return [String]
+    class << self
```

Comment:
Same comments as above re: service object design and moving the method docs down a line.